public/app/plugins/panel/geomap/module.tsx

```diff
@@ -9,6 +9,8 @@ import { mapPanelChangedHandler } from './migrations';
 import { defaultGrafanaThemedMap } from './layers/basemaps';
 import { MARKERS_LAYER_ID } from './layers/data/markersLayer';
 
+const settings = (window as any).grafanaBootData.settings;
```

Comment:
```suggestion
import { config } from 'app/core/config';
....
config.geomapDisableCustomBaseLayer;
```

and add the types to:
https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/config.ts#L89